infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -11,11 +11,17 @@ x-webmaster: &webmaster-release-image-source
   dpl-cms-release: "2024.50.0"
   moduletest-dpl-cms-release: "2025.4.0"
 x-webmasters-on-weekly-release-cycle: &webmasters-on-weekly-release-cycle
+  #This release cycle releases the newest release, which has been tested on webmaster moduletests in the previous week, on to production
```

Comment:
This comment seems a misleading. A wee bit copy/paste here? :smile:

Reply:
just a wee bit :sweat_smile: